### ISSUE TITLE
[ BUG ? ] Repository can't create new project

### ISSUE THREAD

**Issue by jeremy-jimmy** [PyGithub/PyGithub#3071]
I have a repository in my GitHub account, I activated the "Projects" section manually and I created a example project from the web client.

Then I tried to create a new project using the PyGithub package using a simple script as below : 

```python
from github import Github
GITHUB_APP = Github("*************")
REPO_APP = GITHUB_APP.get_repo("*************")

REPO_APP.create_project(name="TEST AUTO PROJET")
```

But I'm getting this error : 

> GithubException: 410 {"message": "Projects are disabled for this repository", "documentation_url": "https://docs.github.com/v3/projects", "status": "410"}

Since, I was able to create a project manually, I'm surprised that it says the projects are disable. It's not the case

Thanks in advance,

**Comment by rikvermeer** [comment]
I have the same, and I've checked the repo settings

**Comment by EnricoMi** [comment]
Documentation says, besides insufficient privileges, `410 Gone` is returned when no classic projects exist in the repository. Looks like (classic) projects can only be added to repositories if thy already have classic projects. Can you confirm your repositories already have classic projects, and that you have successfully added a classic project manually?

https://docs.github.com/en/rest/projects/projects?apiVersion=2022-11-28#create-a-repository-project

**Comment by jeremy-jimmy** [comment]
> Documentation says, besides insufficient privileges, `410 Gone` is returned when no classic projects exist in the repository. Looks like (classic) projects can only be added to repositories if thy already have classic projects. Can you confirm your repositories already have classic projects, and that you have successfully added a classic project manually?
> 
> https://docs.github.com/en/rest/projects/projects?apiVersion=2022-11-28#create-a-repository-project

Yes, I confirm you I was able to create manually a project. What do you mean by classic ? When I created it manually, the "classic" term was never mentioned on the web client.

**Comment by jeremy-jimmy** [comment]
To precise my case, I'm working with private repositories
